### PR TITLE
Milestone 1: The sign up page is finished

### PR DESCRIPTION
**Pull request by RaminMammadzada**
In this milestone we:
- created the signup button.
- created icons for apps.
- created ser form.
- created footer part of the page.
- added the Github Actions linter.

### CODE REVIEW

**Comment by peterrobert** [comment]
### PROJECT UNDER REVIEW BY PETER ROBERT(TSE)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+			<div id="ius-sign-up-copyright" class="ius-hostedui-footer-row ius-hostedui-footer-copyright">
+				<span id="ius-sign-up-copy-container"><span id="ius-sign-up-copy-message">© 2020 Intuit, Inc. All rights
+						reserved. Intuit, QuickBooks, QB, TurboTax, ProConnect and Mint are registered trademarks of
+						Intuit Inc.</span>
+					<br> Terms and conditions, features, support, pricing, and service options subject to change without
+					notice.</span>
+				<br>
+			</div>
+		</div>
+	</footer>
```
Comment (peterrobert) [inline index.html]: - [ ] Kindly position the footer well. It should be below the form check the clone [here](https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26ivid%3D4a885530-b000-4cdf-a17c-2d54f4c5af9d%26adobe_mc%3DMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1591349552%26ivid%3D4a885530-b000-4cdf-a17c-2d54f4c5af9d)
 In your project its a bit off. 
form foote

File: index.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+					<input type="Password" id="Password" name="Password"><br>
+					<input type="submit" value="Create Account">
+				</form>
+				<h5 id="term-and-conditions">
+					<em> By clicking Create account, you agree to our
+						<a href="https://www.mint.com/terms">Terms</a> and
+						have read and acknowledge our <a href="https://www.mint.com/terms">Privacy</a> Statement.</em>
+				</h5>
+			</div>
+		</div>
```
Comment (peterrobert) [inline index.html]: - [ ] Kindly make the form look similar to the [clone](https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26ivid%3D4a885530-b000-4cdf-a17c-2d54f4c5af9d%26adobe_mc%3DMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1591349552%26ivid%3D4a885530-b000-4cdf-a17c-2d54f4c5af9d)
in your project the form is shrinked 
shrink

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: 
@RaminMammadzada AND @Rocio01  **GREAT PROGRESS ** :clap: 

Very impressive work :clap:  You have a descriptive readme file :+1:  but needs a little improvement. You have a descriptive pull request :+1:  but needs a bit of improvement.

However, there are somethings in the project that are preventing me from approving this project kindly make the changes on the following requirements

- [ ] You need to add a screenshot of the exact display of your project. It seems the screenshot you added in the readme file does not resemble your work kindly rectify that.

- [ ] You need to add a header to your pull request messages like described [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)

- [ ] Your HTML file. It has some errors please validate and fix the errors shown.
fix html error

Kindly resubmit after you have done the rectifications :smile: 

**HAPPY CODING** :computer:

**Comment by voscarmv** [comment]
# Pull Request Under Review by Oscar Mier (TSE) 👨‍💻 🐦

**Review by voscarmv — APPROVED** [review]
# Excellent work applying the requested changes team! It's a ✔️Yes ✔️ for me! 😄 👍 

**[OPTIONAL]** I strongly recommend you fix the *Live Demo Link* on your `README.md` file! 👀 

Potential employers might be watching, and you definitely want them to be able to look at your work!

**[OPTIONAL]** Try using fonts that are similar to the original. It's not required, but it will give potential employers a better impression if they come around to look at your github! 👨‍💼 👩‍💼 

Other than that, it's all good 👍 

# Comments? Questions? Contact me on [Twitter](http://www.twitter.com/voscarmv) or [LinkedIn](http://www.linkedin.com/in/oscar-mier) 🧑‍💻 🗨️ 🧑‍💻 

## Happy Coding! 🧠 ⌨️ 💻

**Pull request merged**